circleci/integration-test

```diff
@@ -93,9 +93,13 @@ download_test_artifacts(){
   for workflowID in $(cat integration-tests.out | jq -r '.testWorkflowIds[]');
   do
     S3_BUCKET_URL="s3://frontend-to-end-tester-files-dev/${workflowID}/artifacts.zip"
+    S3_BUCKET="frontend-to-end-tester-files-dev"
+    S3_ARTIFACTS_KEY="${workflowID}/artifacts.zip"
     # TODO we probably at some point want to separate out the integration test script from the artifacts download
     # that way when we want to use i-t-s for other tests besides fe2e, we can still use this script
-    if ! aws s3 ls $S3_BUCKET_URL >/dev/null 2>&1; then
+    NOT_EXIST=$(aws s3api head-object --profile ${AWS_S3_PROFILE} --bucket ${S3_BUCKET} --key ${S3_ARTIFACTS_KEY} >/dev/null 2>1; echo $?)
+    # 254 is the code that the s3api returns for when the object does not exist
```

Comment:
Found documentation for code 254 at https://awscli.amazonaws.com/v2/documentation/api/latest/topic/return-codes.html